src/ui/SettingsButton.tsx

```diff
@@ -43,18 +43,19 @@ export default function SettingsCard({ navigator }: { navigator: Navigator }) {
     <>
       <Popover
         placement="bottom-end"
-        closeOnBlur={false}
         isOpen={isOpen}
         onClose={close}
-        autoFocus={false}
+        autoFocus={true}
       >
         <PopoverTrigger>
           <Button onClick={open}>Settings</Button>
         </PopoverTrigger>
         <PopoverContent
           borderColor="gray.100"
           width="fit-content"
-          _active={{ boxShadow: 'none' }}
+          _active={{ boxShadow: 'lg' }}
+          _focus={{ boxShadow: 'lg' }}
+          boxShadow="lg"
```

Comment:
Do you need to have this on `_active` and `_focus`? If you just have it without the pseudoselectors does it not display correctly?

Reply:
I believe we do need to explicitly define them if we want to be consistent with the boxShadow style. The default one was a blurred blue border.

Reply:
I think we shouldn't ever remove the focus ring, if the element is focused it should have the focus ring. There are times when we can remove it if the user isn't actively tabbing through the site, but we shouldn't get that fancy yet. 

I think we should set the initial focus when opening the popover to the first choice the user has. You can see an example of that here:

https://chakra-ui.com/docs/overlay/popover#focus-an-element-when-popover-opens